google-cloud-graalvm-samples/google-cloud-graalvm-pubsub-sample/src/main/java/com/example/Driver.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.pubsub.v1.Publisher;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+
+public class Driver {
+
+  /**
+   * Driver for the Sample application compiled with GraalVM which uses Pub/Sub client libraries.
+   */
+  public static void main(String[] args) throws Exception {
+    Publisher publisher =
+        Publisher
+            .newBuilder("projects/my-kubernetes-codelab-217414/topics/exampleTopic")
```

Comment:
Externalize topic name into a property?

Reply:
+1

Reply:
I made it into a command line arg, done.